src/main/zapHomeFiles/hud/tools/toggleScript.js

```diff
@@ -0,0 +1,190 @@
+/*
+ * HTML Report tool
+ *
+ * When selected displays the standard HTML report in a new window / tab
+ */
+
+const ToggleScript = (function () {
+	// Constants
+	const NAME = 'toggleScript';
+	const LABEL = 'Toggle Script';
+	const ICON = {};
+	ICON.default = 'script-add.png';
+	ICON.on = 'script-enabled.png';
+	ICON.off = 'script-disabled.png';
+	let SCRIPT;
+
+	function initializeStorage() {
+		const tool = {};
+		tool.name = NAME;
+		tool.data = '';
+		tool.panel = '';
+		tool.position = 0;
+		tool.label = LABEL;
+		tool.icon = ICON.default;
+		utils.writeTool(tool);
+	}
+
+	// Select a script and update the UI accordingly
+	function showOptions(tabId) {
+		// Pop up a menu to select a script
+		selectScript(tabId)
+			.then(result => {
+				if (!result.remove) {
+					SCRIPT = result.selected;
+					updateUI();
+				} else {
+					// Set the currently selected script to null
```

Comment:
I'd suggest removing these comments, (IMO) they just repeat what the code is doing.